From b42590882732bd0134e1ee0d928723a8e70666b4 Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Mon, 19 Nov 2018 12:15:55 +0000
Subject: [PATCH 51/52] Fix case of WebKit/WebKit.h (was Webkit/WebKit.h)

If you build on a case-sensitive FS on macOS this is needed.
---
 src/cpp/desktop-mac/WebViewController.h  | 2 +-
 src/cpp/desktop-mac/WebViewController.mm | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/cpp/desktop-mac/WebViewController.h b/src/cpp/desktop-mac/WebViewController.h
index ab39bcd210..55153b8d83 100644
--- a/src/cpp/desktop-mac/WebViewController.h
+++ b/src/cpp/desktop-mac/WebViewController.h
@@ -1,7 +1,7 @@
 
 
 #import <Cocoa/Cocoa.h>
-#import <Webkit/WebKit.h>
+#import <WebKit/WebKit.h>
 
 #import "WebViewWithKeyEquiv.h"
 #import "GwtCallbacks.h"
diff --git a/src/cpp/desktop-mac/WebViewController.mm b/src/cpp/desktop-mac/WebViewController.mm
index 7a52921315..ef6f9bebab 100644
--- a/src/cpp/desktop-mac/WebViewController.mm
+++ b/src/cpp/desktop-mac/WebViewController.mm
@@ -7,7 +7,7 @@
 #import <Cocoa/Cocoa.h>
 
 #import <WebKit/WebFrame.h>
-#import <Webkit/WebUIDelegate.h>
+#import <WebKit/WebUIDelegate.h>
 
 #import "Options.hpp"
 #import "WebViewController.h"
-- 
2.19.1

